lib/containers/engine.pm

```diff
@@ -325,6 +327,9 @@ sub cleanup_system_host {
     # all containers should be stopped before running prune
     # https://github.com/containers/podman/issues/19038
     if ($self->runtime eq 'podman') {
+        if (package_version_cmp(get_podman_version(), '2.1.1') == 0) {
```

Comment:
I have changed it now for less than 4.0.0. As we have 2 major versions out there

Reply:
> Yes, I have changed it now for less than 4.0.0. As we have 2 major versions out there

I believe the comparation should be against 2.1.1 in case podman in older SP's is updated to a version < 4.0.0.

Reply:
2.1.1 is currently used in SP1 and SP2, I assume that is the oldest podman version that we use. So in case there will be a minor version bump in LTSS we would need to again update the code.

Reply:
> 2.1.1 is currently used in SP1 and SP2, I assume that is the oldest podman version that we use. So in case there will be a minor version bump in LTSS we would need to again update the code.

Agree.